libsolidity/ast/AST.h

```diff
@@ -619,11 +621,13 @@ class CallableDeclaration: public Declaration, public VariableScope
 	std::vector<ASTPointer<VariableDeclaration>> const& returnParameters() const { return m_returnParameters->parameters(); }
 	ParameterList const& parameterList() const { return *m_parameters; }
 	ASTPointer<ParameterList> const& returnParameterList() const { return m_returnParameters; }
+	bool isVirtual() const { return m_isVirtual; }
 
 protected:
 	ASTPointer<ParameterList> m_parameters;
 	ASTPointer<OverrideSpecifier> m_overrides;
 	ASTPointer<ParameterList> m_returnParameters;
+	bool m_isVirtual;
```

Comment:
Member initialize to ``false`` (yeah, it's redundant, but still better to do it and we said we'd want to always do it for PODs).